src/test/java/no/nav/veilarbvedtaksstotte/config/ClientTestConfig.java

```diff
@@ -112,8 +113,8 @@ public HealthCheckResult checkHealth() {
     public VeilarbarenaClient arenaClient() {
         return new VeilarbarenaClient() {
             @Override
-            public EnhetId oppfolgingsenhet(Fnr fnr) {
-                return EnhetId.of(TEST_OPPFOLGINGSENHET_ID);
+            public VeilarbArenaOppfolging hentOppfolgingsbruker(Fnr fnr) {
+                return new VeilarbArenaOppfolging(TEST_OPPFOLGINGSENHET_ID, "ISERV");
```

Comment:
Kanskje ikke ISERV bør være defaulten her for evt Spring Boot tester/kjøre lokalt?

Reply:
Enig i det 👍